source/appModules/powerpnt.py

```diff
@@ -407,18 +407,18 @@ def _get_firstChild(self):
 
 	def handleSelectionChange(self):
 		"""Pushes focus to the newly selected object."""
-		if getattr(self,"_isHandlingSelectionChange",False):
-			# #3394: A COM event can cause this function to run within itself.
+		if getattr(self, "_isHandlingSelectionChange", False):

```

Comment:
There are still some whitespace changes here that are distracting from the change.

Reply:
ah that was my editor changing LF to CRLF and then causing the lint to fail. I've fixed this now